mlir/python/mlir/rewrite.py

```diff
@@ -0,0 +1,5 @@
+#  Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+#  See https://llvm.org/LICENSE.txt for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from ._mlir_libs._mlir.rewrite import *
```

Comment:
I think this will work

```suggestion
from ._mlir_libs._mlir.rewrite import *
from .extras.meta import region_op

pattern = region_op(PatternOp)
rewrite = region_op(RewriteOp)
```

Reply:
I ended up doing this inside pdl instead.